### PR TITLE
Removing Payouts error

### PR DESCRIPTION
**Pull request by aninja8**
This PR contains changes to remove the Payouts Error object to simplify the integration

### CODE REVIEW

File: PayPalPayoutsSdk/PayPalPayoutsSdk.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageVersion>1.0.3</PackageVersion>
+    <PackageVersion>1.0.1</PackageVersion>
```
Comment (rajkumarpalani) [inline PayPalPayoutsSdk/PayPalPayoutsSdk.csproj:5]: this is not a patch fix, can this be treated as a minor version

File: PayPalPayoutsSdk/Payouts/PayoutItemResponse.cs [hunk 2]
```diff
@@ -33,12 +33,6 @@ public class PayoutItemResponse
         [DataMember(Name="currency_conversion", EmitDefaultValue = false)]
         public PayoutCurrencyConversion CurrencyConversion;
 
-        /// <summary>
-        /// The error details.
-        /// </summary>
-        [DataMember(Name="errors", EmitDefaultValue = false)]
-        public PayoutError Errors;
```
Comment (rajkumarpalani) [inline PayPalPayoutsSdk/Payouts/PayoutItemResponse.cs:40]: this has to be replaced with Error to represent the item failure details

**Review by rajkumarpalani — CHANGES_REQUESTED** [review]
paypal client needs to be updated with the version change for fpti header
ErrorDetails needs to be added with location 
PayoutError to be removed from PayoutBatch
Error to be removed from PayoutBatchHeader
Error to be removed from PayoutHeader

File: PayPalPayoutsSdk/PayPalPayoutsSdk.csproj (hunk 1 above)
Comment (aninja8) [reply PayPalPayoutsSdk/PayPalPayoutsSdk.csproj]: changed

File: PayPalPayoutsSdk/Payouts/PayoutItemResponse.cs (hunk 2 above)
Comment (aninja8) [reply PayPalPayoutsSdk/Payouts/PayoutItemResponse.cs:40]: done

File: CHANGE.ME [hunk 3]
```diff
@@ -0,0 +1,6 @@
+### 1.1.0 (2021-02-24)
```
Comment (rajkumarpalani) [inline CHANGE.ME:1]: remove this file

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+### 1.1.0 (2021-02-24)
+
+*  Remove unused error from CreatePayoutResponse [https://github.com/paypal/Payouts-DotNet-SDK/pull/7/commits/9abd12cfbe35097e600500952f0a445295e3ea80]
```
Comment (rajkumarpalani) [inline CHANGELOG.md:3]: add a hyperlink to commit short id
refer commit directly, avoid pull request reference to commit

File: Samples/CancelPayoutItemSample.cs [hunk 5]
```diff
@@ -35,9 +28,16 @@ public async static Task<HttpResponse> CancelItem(string itemId,bool debug = fal
...
+               
+                String errorString = ex.Message;
+                Error error= ErrorUtil.deserializeError(errorString);
+                ErrorUtil.printError(error);
+                return null;
+            }
         }
 
-        static void Main(string[] args)
+        /*static void Main(string[] args)
```
Comment (rajkumarpalani) [inline Samples/CancelPayoutItemSample.cs:40]: is it required to comment this code?

File: Test/Payouts/PayoutsGetTest.cs [hunk 6]
```diff
@@ -33,5 +34,34 @@ public async void TestPayoutsGetRequest()
...
+
+            PayoutsGetRequest request = new PayoutsGetRequest("123")
+                .Page(4)
+                .PageSize(2)
+                .TotalRequired(true);
+
+            
+            try {
+                HttpResponse getResponse = await TestHarness.client().Execute(request);
+            } catch(Exception ex){
```
Comment (rajkumarpalani) [inline Test/Payouts/PayoutsGetTest.cs:50]: may fail when there is a non HttpException raised

**Review by rajkumarpalani — APPROVED** [review]
Looks good
Suggested some minor changes to changelog, tests

File: CHANGE.ME (hunk 3 above)
Comment (aninja8) [reply CHANGE.ME]: done

File: CHANGELOG.md (hunk 4 above)
Comment (aninja8) [reply CHANGELOG.md]: done

File: Samples/CancelPayoutItemSample.cs (hunk 5 above)
Comment (aninja8) [reply Samples/CancelPayoutItemSample.cs]: removed

File: Test/Payouts/PayoutsGetTest.cs (hunk 6 above)
Comment (aninja8) [reply Test/Payouts/PayoutsGetTest.cs]: updated

**Review by ZRDKPoWeR — APPROVED** [review]

**Review by joshuaslaney — COMMENTED** [review]
Slaneyjosh14@gmail.com

**Pull request merged**